libs/sdk-backend-tiger/src/backend/workspace/exportDefinitions/index.ts

```diff
@@ -0,0 +1,184 @@
+import { objRefToIdentifier } from "../../../utils/api.js";
+
+import { InMemoryPaging } from "@gooddata/sdk-backend-base";
+import {
+    exportDefinitionOutDocumentToExportDefinition,
+    exportDefinitionOutDocumentToExportDefinitionOutWithLinks,
+    exportDefinitionOutToExportDefinition,
+    exportDefinitionToExportDefinitionInDocument,
+    exportDefinitionToExportDefinitionPostOptionalIdDocument,
+} from "../../../convertors/fromBackend/ExportDefinitionsConverter.js";
+import { ExportDefinitionsQuery } from "./exportDefinitionsQuery.js";
+import { exportDefinitionsListComparator } from "./comparator.js";
+
+export class TigerWorkspaceExportDefinitions implements IWorkspaceExportDefinitionsService {
+    constructor(private readonly authCall: TigerAuthenticatedCallGuard, public readonly workspace: string) {}
+
+    public getExportDefinitions = async (
+        options?: IExportDefinitionsQueryOptions,
+    ): Promise<IExportDefinitionsQueryResult> => {
+        const requestParameters = this.getExportDefinitionsRequestParameters(options);
+        const allExportDefinitions = await this.authCall((client) => {
+            return MetadataUtilities.getAllPagesOf(
+                client,
+                client.entities.getAllEntitiesExportDefinitions,
+                requestParameters,
+                { headers: ValidateRelationsHeader },
+            )
+                .then(MetadataUtilities.mergeEntitiesResults)
+                .then(MetadataUtilities.filterValidEntities)
+                .then((res) => {
+                    if (options?.title) {
+                        const lowercaseSearch = options.title.toLocaleLowerCase();
+
+                        return res.data
+                            .filter((ed) => {
+                                const title = ed.attributes?.title;
+
+                                return title && title.toLowerCase().indexOf(lowercaseSearch) > -1;
+                            })
+                            .map((ed) => exportDefinitionOutToExportDefinition(ed, res.included));
+                    }
+                    return res.data.map((ep) => exportDefinitionOutToExportDefinition(ep, res.included));
+                });
+        });
+
+        // Remove when API starts to support sort=modifiedBy,createdBy,insight.title
+        // (first verify that modifiedBy,createdBy behave as the code below, i.e., use createdBy if modifiedBy is
+        // not defined as it is missing for the insights that were just created and never updated, also title
+        // should be compared in case-insensitive manner)
+
+        const sanitizedOrder =
+            requestParameters.sort === undefined && allExportDefinitions.length > 0
+                ? [...allExportDefinitions].sort(exportDefinitionsListComparator)
+                : allExportDefinitions;
+
+        return new InMemoryPaging(sanitizedOrder, options?.limit ?? 50, options?.offset ?? 0);
```

Comment:
This is problematic ... Why needs to be sorted as is? And do in memory paging ? 

When this will fetch bigger amount data this leads to same problem as 

This leads to same problem asi this task 


https://gooddata.atlassian.net/browse/STL-369

Reply:
As discussed on a call with @ivanmjartan and @pbenes, for now I added a comment about the InMemoryPaging being a temporary solution, and we will address the issue in https://gooddata.atlassian.net/browse/STL-397 once the ServerPaging is implemented in the insights service.